features/networking/sctp.feature

```diff
@@ -5,13 +5,14 @@ Feature: SCTP related scenarios
   @admin
   @destructive
   Scenario: Establish pod to pod SCTP connections
-    Given I store the workers in the :workers clipboard
-    Given I install machineconfigs load-sctp-module
-    Given I have a project
+    Given I store the ready and schedulable workers in the :workers clipboard
```

Comment:
hmm, this step can make sure we have 2 or more schedulable workers? I think it's only store the ready and schedulable worker.

Reply:
@zhaozhanqi https://github.com/openshift/verification-tests/pull/1823/files#diff-7539dbdee665a3ee90dc338a1a8258f9c4723c11850769dfbf6216c6e55a5f42R1178 will make sure the cluster will have more than one ready and schedulable worker.

Reply:
ah, ok